extensions/Cake.Xrm.Sdk.yml

```diff
@@ -0,0 +1,17 @@
+Type: Addin
+Name: Cake.Xrm.Sdk
+NuGet: Cake.Xrm.Sdk
```

Comment:
Repo doesn't seem to be anywhere on GitHub... Blacklist

Reply:
Adding this addin to exclusion list: : https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/194